### ISSUE TITLE
Issue with v0.84 and v0.85

### ISSUE THREAD

**Issue by bliatun** [ebaauw/homebridge-hue#311]
#### Issue

0.83 currently working fine. 
0.84 Unable to start Homebridge 
0.85 Homebridge runs, but Homebridge-hue fails

#### Log Messages

```
[2018-3-12 16:23:07] [Hue] Philips hue: BSB002 bridge v1801260942, api v1.23.0
[2018-3-12 16:23:08] [Hue] Philips hue: error: /groups/15: {"name":"Taklampe Soverommet","lights":["14","15"],"type":"LightGroup","state":{"all_on":true,"any_on":true,"bri":127,"ct":250,"alert":"none","colormode":"xy"},"recycle":false,"action":{"on":true,"bri":127,"ct":250,"alert":"none","colormode":"xy"}}
TypeError: Cannot read property 'isMulti' of undefined
at HueLight.setConfig (/usr/local/lib/node_modules/homebridge-hue/lib/HueLight.js:508:23)
at new HueLight (/usr/local/lib/node_modules/homebridge-hue/lib/HueLight.js:421:8)
at HueAccessory.exposeGroupResource (/usr/local/lib/node_modules/homebridge-hue/lib/HueAccessory.js:148:19)
at HueAccessory.exposeGroupResources (/usr/local/lib/node_modules/homebridge-hue/lib/HueAccessory.js:139:10)
at HueAccessory.expose (/usr/local/lib/node_modules/homebridge-hue/lib/HueAccessory.js:121:8)
at HueBridge.exposeResources (/usr/local/lib/node_modules/homebridge-hue/lib/HueBridge.js:463:49)
at getConfig.then.then.then.then (/usr/local/lib/node_modules/homebridge-hue/lib/HueBridge.js:92:19)
at Function.then (/usr/local/lib/node_modules/homebridge-hue/node_modules/deferred/_ext.js:67:13)
at Deferred._settle (/usr/local/lib/node_modules/homebridge-hue/node_modules/deferred/deferred.js:97:26)
at Deferred.resolve (/usr/local/lib/node_modules/homebridge-hue/node_modules/deferred/deferred.js:142:15)
at request.then (/usr/local/lib/node_modules/homebridge-hue/lib/HueBridge.js:295:11)
at Function.then (/usr/local/lib/node_modules/homebridge-hue/node_modules/deferred/_ext.js:67:13)
at Deferred._settle (/usr/local/lib/node_modules/homebridge-hue/node_modules/deferred/deferred.js:97:26)
at Deferred.resolve (/usr/local/lib/node_modules/homebridge-hue/node_modules/deferred/deferred.js:142:15)
at Request.request [as _callback] (/usr/local/lib/node_modules/homebridge-hue/lib/HueBridge.js:1098:14)
at Request.self.callback (/usr/local/lib/node_modules/homebridge-hue/node_modules/request/request.js:186:22)
at emitTwo (events.js:126:13)
at Request.emit (events.js:214:7)
at Request.<anonymous> (/usr/local/lib/node_modules/homebridge-hue/node_modules/request/request.js:1163:10)
at emitOne (events.js:116:13)
at Request.emit (events.js:211:7)
at IncomingMessage.<anonymous> (/usr/local/lib/node_modules/homebridge-hue/node_modules/request/request.js:1085:12)
at Object.onceWrapper (events.js:313:30)
at emitNone (events.js:111:20)
at IncomingMessage.emit (events.js:208:7)
at endReadableNT (_stream_readable.js:1055:12)
at _combinedTickCallback (internal/process/next_tick.js:138:11)
at process._tickCallback (internal/process/next_tick.js:180:9)
[2018-3-12 16:23:08] [Hue] Philips hue: error: /groups/17: {"name":"Taklampe Hobbyrommet ","lights":["29","30"],"type":"LightGroup","state":{"all_on":false,"any_on":false,"bri":126,"alert":"none"},"recycle":false,"action":{"on":false,"bri":126,"alert":"none"}}
TypeError: Cannot read property 'isMulti' of undefined
at HueLight.setConfig (/usr/local/lib/node_modules/homebridge-hue/lib/HueLight.js:508:23)
at new HueLight (/usr/local/lib/node_modules/homebridge-hue/lib/HueLight.js:421:8)
at HueAccessory.exposeGroupResource (/usr/local/lib/node_modules/homebridge-hue/lib/HueAccessory.js:148:19)
at HueAccessory.exposeGroupResources (/usr/local/lib/node_modules/homebridge-hue/lib/HueAccessory.js:139:10)
at HueAccessory.expose (/usr/local/lib/node_modules/homebridge-hue/lib/HueAccessory.js:121:8)
at HueBridge.exposeResources (/usr/local/lib/node_modules/homebridge-hue/lib/HueBridge.js:463:49)
at getConfig.then.then.then.then (/usr/local/lib/node_modules/homebridge-hue/lib/HueBridge.js:92:19)
at Function.then (/usr/local/lib/node_modules/homebridge-hue/node_modules/deferred/_ext.js:67:13)
at Deferred._settle (/usr/local/lib/node_modules/homebridge-hue/node_modules/deferred/deferred.js:97:26)
at Deferred.resolve (/usr/local/lib/node_modules/homebridge-hue/node_modules/deferred/deferred.js:142:15)
at request.then (/usr/local/lib/node_modules/homebridge-hue/lib/HueBridge.js:295:11)
at Function.then (/usr/local/lib/node_modules/homebridge-hue/node_modules/deferred/_ext.js:67:13)
at Deferred._settle (/usr/local/lib/node_modules/homebridge-hue/node_modules/deferred/deferred.js:97:26)
at Deferred.resolve (/usr/local/lib/node_modules/homebridge-hue/node_modules/deferred/deferred.js:142:15)
at Request.request [as _callback] (/usr/local/lib/node_modules/homebridge-hue/lib/HueBridge.js:1098:14)
at Request.self.callback (/usr/local/lib/node_modules/homebridge-hue/node_modules/request/request.js:186:22)
at emitTwo (events.js:126:13)
at Request.emit (events.js:214:7)
at Request.<anonymous> (/usr/local/lib/node_modules/homebridge-hue/node_modules/request/request.js:1163:10)
at emitOne (events.js:116:13)
at Request.emit (events.js:211:7)
at IncomingMessage.<anonymous> (/usr/local/lib/node_modules/homebridge-hue/node_modules/request/request.js:1085:12)
at Object.onceWrapper (events.js:313:30)
at emitNone (events.js:111:20)
at IncomingMessage.emit (events.js:208:7)
at endReadableNT (_stream_readable.js:1055:12)
at _combinedTickCallback (internal/process/next_tick.js:138:11)
at process._tickCallback (internal/process/next_tick.js:180:9)
[2018-3-12 16:23:08] [Hue] Philips hue: error: /groups/18: {"name":"Utelys","lights":["34","21","23","33","22"],"type":"LightGroup","state":{"all_on":false,"any_on":false,"bri":178,"alert":"none"},"recycle":false,"action":{"on":false,"bri":178,"alert":"none"}}
TypeError: Cannot read property 'isMulti' of undefined
at HueLight.setConfig (/usr/local/lib/node_modules/homebridge-hue/lib/HueLight.js:508:23)
at new HueLight (/usr/local/lib/node_modules/homebridge-hue/lib/HueLight.js:421:8)
at HueAccessory.exposeGroupResource (/usr/local/lib/node_modules/homebridge-hue/lib/HueAccessory.js:148:19)
at HueAccessory.exposeGroupResources (/usr/local/lib/node_modules/homebridge-hue/lib/HueAccessory.js:139:10)
at HueAccessory.expose (/usr/local/lib/node_modules/homebridge-hue/lib/HueAccessory.js:121:8)
at HueBridge.exposeResources (/usr/local/lib/node_modules/homebridge-hue/lib/HueBridge.js:463:49)
at getConfig.then.then.then.then (/usr/local/lib/node_modules/homebridge-hue/lib/HueBridge.js:92:19)
at Function.then (/usr/local/lib/node_modules/homebridge-hue/node_modules/deferred/_ext.js:67:13)
at Deferred._settle (/usr/local/lib/node_modules/homebridge-hue/node_modules/deferred/deferred.js:97:26)
at Deferred.resolve (/usr/local/lib/node_modules/homebridge-hue/node_modules/deferred/deferred.js:142:15)
at request.then (/usr/local/lib/node_modules/homebridge-hue/lib/HueBridge.js:295:11)
at Function.then (/usr/local/lib/node_modules/homebridge-hue/node_modules/deferred/_ext.js:67:13)
at Deferred._settle (/usr/local/lib/node_modules/homebridge-hue/node_modules/deferred/deferred.js:97:26)
at Deferred.resolve (/usr/local/lib/node_modules/homebridge-hue/node_modules/deferred/deferred.js:142:15)
at Request.request [as _callback] (/usr/local/lib/node_modules/homebridge-hue/lib/HueBridge.js:1098:14)
at Request.self.callback (/usr/local/lib/node_modules/homebridge-hue/node_modules/request/request.js:186:22)
at emitTwo (events.js:126:13)
at Request.emit (events.js:214:7)
at Request.<anonymous> (/usr/local/lib/node_modules/homebridge-hue/node_modules/request/request.js:1163:10)
at emitOne (events.js:116:13)
at Request.emit (events.js:211:7)
at IncomingMessage.<anonymous> (/usr/local/lib/node_modules/homebridge-hue/node_modules/request/request.js:1085:12)
at Object.onceWrapper (events.js:313:30)
at emitNone (events.js:111:20)
at IncomingMessage.emit (events.js:208:7)
at endReadableNT (_stream_readable.js:1055:12)
at _combinedTickCallback (internal/process/next_tick.js:138:11)
at process._tickCallback (internal/process/next_tick.js:180:9)
[2018-3-12 16:23:08] [Hue] Philips hue: 4 accessories
[2018-3-12 16:23:08] [Hue] masked debug info dumped to /var/homebridge/homebridge-hue.json.gz
[2018-3-12 16:23:08] [Hue] Initializing platform accessory 'Philips hue'...
[2018-3-12 16:23:08] [Hue] Philips hue: 3 services
[2018-3-12 16:23:08] [Hue] Initializing platform accessory 'Taklampe Soverommet'...
[2018-3-12 16:23:08] [Hue] TypeError: Cannot read property 'sType' of undefined
[homebridge-hue.json.gz](https://github.com/ebaauw/homebridge-hue/files/1803857/homebridge-hue.json.gz)

```

**Comment by ebaauw** [comment]
I take it this is the v0.8.5 log?  I'm afraid that's due to an embarrassing typo.

What was the error in v0.8.4?  Is it the same as #306?

**Comment by bliatun** [comment]
Yes, log from v.0.8.5  Errors in v.0.8.4 looked the same.

**Comment by Krocko** [comment]
Is this the same error?
```
Mar 11 16:21:41 raspberrypi homebridge[7454]: [2018-3-11 16:21:41] [Hue] deCONZ-GW: TypeError: Cannot read property 'split' of undefined
Mar 11 16:21:41 raspberrypi homebridge[7454]: at HueBridge.exposeSensor (/usr/local/lib/node_modules/homebridge-hue/lib/HueBridge.js:545:20)
Mar 11 16:21:41 raspberrypi homebridge[7454]: at HueBridge.exposeSensors (/usr/local/lib/node_modules/homebridge-hue/lib/HueBridge.js:518:14)
Mar 11 16:21:41 raspberrypi homebridge[7454]: at HueBridge.exposeResources (/usr/local/lib/node_modules/homebridge-hue/lib/HueBridge.js:453:8)
Mar 11 16:21:41 raspberrypi homebridge[7454]: at getConfig.then.then.then.then (/usr/local/lib/node_modules/homebridge-hue/lib/HueBridge.js:92:19)
Mar 11 16:21:41 raspberrypi homebridge[7454]: at Function.then (/usr/local/lib/node_modules/homebridge-hue/node_modules/deferred/_ext.js:67:13)
Mar 11 16:21:41 raspberrypi homebridge[7454]: at Deferred._settle (/usr/local/lib/node_modules/homebridge-hue/node_modules/deferred/deferred.js:97:26)
Mar 11 16:21:41 raspberrypi homebridge[7454]: at Deferred.resolve (/usr/local/lib/node_modules/homebridge-hue/node_modules/deferred/deferred.js:142:15)
Mar 11 16:21:41 raspberrypi homebridge[7454]: at request.then (/usr/local/lib/node_modules/homebridge-hue/lib/HueBridge.js:295:11)
Mar 11 16:21:41 raspberrypi homebridge[7454]: at Function.then (/usr/local/lib/node_modules/homebridge-hue/node_modules/deferred/_ext.js:67:13)
Mar 11 16:21:41 raspberrypi homebridge[7454]: at Deferred._settle (/usr/local/lib/node_modules/homebridge-hue/node_modules/deferred/deferred.js:97:26)
Mar 11 16:21:41 raspberrypi homebridge[7454]: at Deferred.resolve (/usr/local/lib/node_modules/homebridge-hue/node_modules/deferred/deferred.js:142:15)
Mar 11 16:21:41 raspberrypi homebridge[7454]: at Request.request [as _callback] (/usr/local/lib/node_modules/homebridge-hue/lib/HueBridge.js:1098:14)
Mar 11 16:21:41 raspberrypi homebridge[7454]: at Request.self.callback (/usr/local/lib/node_modules/homebridge-hue/node_modules/request/request.js:186:22)
Mar 11 16:21:41 raspberrypi homebridge[7454]: at Request.emit (events.js:180:13)
Mar 11 16:21:41 raspberrypi homebridge[7454]: at Request.<anonymous> (/usr/local/lib/node_modules/homebridge-hue/node_modules/request/request.js:1163:10)
Mar 11 16:21:41 raspberrypi homebridge[7454]: at Request.emit (events.js:180:13)
Mar 11 16:21:41 raspberrypi homebridge[7454]: at IncomingMessage.<anonymous> (/usr/local/lib/node_modules/homebridge-hue/node_modules/request/request.js:1085:12)
Mar 11 16:21:41 raspberrypi homebridge[7454]: at Object.onceWrapper (events.js:272:13)
Mar 11 16:21:41 raspberrypi homebridge[7454]: at IncomingMessage.emit (events.js:185:15)
Mar 11 16:21:41 raspberrypi homebridge[7454]: at endReadableNT (_stream_readable.js:1101:12)
Mar 11 16:21:41 raspberrypi homebridge[7454]: at process._tickCallback (internal/process/next_tick.js:114:19)
```

**Comment by ebaauw** [comment]
@Krocko No, it's not.  Can you please post the full debug log file (see https://github.com/ebaauw/homebridge-hue#debug-log-file) and dump file (see https://github.com/ebaauw/homebridge-hue#debug-dump-file)?

**Comment by Krocko** [comment]
Sorry, i can‘t. It was yesterday and i have restartet several times. 
I can tell you how to reproduce this. 😄
I have opend  the zll.db from deCONZ with the SQLiteBrowser and changed a ClipGenericStatus Sensor. "manufacturername" to "homebridge-hue" and leave the "uniqueid" emty.

**Comment by ebaauw** [comment]
OK, I'm not going to handle that - `uniqueid` is a required string 6..32, so it shouldn't be empty.

**Comment by Krocko** [comment]
Ok. The uniqueid from the deCONZ daylight Sensor is emty too. So i testet this for e ClipGenericStatus Sensor. This works for the ClipGenericStatus Sensor too, but the "manufacturername" must be something different as "homebridge-hue".

**Comment by ebaauw** [comment]
The Daylight sensor doesn't have a `uniqueid` in the REST API.  And is has manufacturername Philips (also for deCONZ).

**Comment by ebaauw** [comment]
Fixed in v0.8.6.

**Comment by bliatun** [comment]
Confirmed working perfectly in v.0.8.6. Thanks for your support!
